### DISCUSSION TITLE
Openvino on integrated graphics

### DISCUSSION THREAD

**Discussion by tye-mustafa** [immich-app/immich#14813] [Q&A] [answered]
as said in documentation that integrated graphics intel should not go for openvino but my question is why and what if i go what would be the performance difference between cpu and openvino library?

using n305 cpu 
RAM: 32GB DDR5

**Comment by mertalev** [comment]
There have been many issues with iGPUs in the past, sometimes causing the model to not work and sometimes "seeming" to work but providing worse results. These issues don't exist right now, but it's very possible for them to happen again in the future. We try to unofficially support iGPUs anyway, but can't provide a guarantee about it since it can be tied to upstream issues we can't resolve on our own.

Regarding performance, it highly depends on your processor, but a typical speedup would be roughly 3x faster.

**Reply by tye-mustafa** [reply, marked as answer]
Have tested for 
N305 intel cpu performance mode 
the results for cpu and openvino are nearly same, but on cpu it consumes more power,

on cpu multiple images can be processed but on gpu only one is processed it if we compare 4 pics given to cpu take 35 secs and gpu process 4 pics in 37 secs nearly same.

same if given 7 images to cpu process it will render faster than gpu but so much power is wasted, im going to permanently use openvino as my main smart search engine.

**Reply by lukasig** [reply]
I also have a N305 intel CPU and for some reason openvino is not working even though i modified my docker-compose.yml file accordingly .
can it conflict with frigate (which is using pcie coral) in any way?
